backend/site-api/src/app/site/create_site.rs

```diff
@@ -44,6 +46,11 @@ pub async fn create_site_helper(
         .await
         .map_err(|e| ApiError::internal_error().message(e))?;
 
+    if let Some(url) = backup_url {
```

Comment:
Due to the backup being on the old server before migration, there are no backup records available after creating the new site. If I call `/sites/:site_id/backups/:backup_id/actions/restore` on the new server, it will fail. 
I have considered copying the old backup record to the new server before calling restore, but I think this approach is not ideal. Instead, I adding `backup_url` to `create_site_dto` as an Option. If the value is `Some`, it indicates that the site to be created comes with default data. 
Let me know if this approach makes sense and if there are any improvements needed!